proposals/lip-0041.md

```diff
@@ -209,7 +209,7 @@ invalidSigDataSchema = {
 
 ### Commands
 
-#### RegisterMultisignatureGroup
+#### RegisterMultisignature
```

Comment:
There are few more other occurrences of 
- "register multisignature group command"
- "register multisignature group transaction"
- "COMMAND_REGISTER_MULTISIGNATURE_GROUP"

that should be changed as well.

Reply:
I removed all instances of 'group'